script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -1089,7 +1091,7 @@ I can feel the palpitations of Arcueid's heart.
 [sha:33be29b32b1f0b1b804f8c5825f53399bfb39148]{
 -- Page 225, Offset 30154.
 -- いまアルクェイドを愛してるなら、それが遠野志貴の真実だ。そのあとの事なんて、知らない。
-If in this moment I love Arcueid, then that's Tohno Shiki's truth.
+If in this moment I love Arcueid, then that's what's true for Tohno Shiki.
```

Comment:
Kinda prefer the original formulation. Maybe I am JP brain poisoned, but "live your truth" is definitely an en phrase.